kernel_tuner/observers/ncu.py

```diff
@@ -0,0 +1,36 @@
+from kernel_tuner.observers import PrologueObserver
+
+try:
+    import nvmetrics
+except (ImportError):
+    nvmetrics = None
+    pass
+
+class NCUObserver(PrologueObserver):
+    """``NCUObserver`` measures performance counters.
+
+    """
+
+    def __init__(self, metrics=None):
+        """Create a new ``NCUObserver``.
+
+        :param metrics: The metrics to observe. This should be a list of strings.
+                        You can use ``ncu --query-metrics`` to get a list of valid metrics.
+        """
+
+        if not nvmetrics:
+            print("NCUObserver is not available.")
```

Comment:
Maybe best to just raise an exception here. It could be confusing to userscreate an `NCUObserver` that just does nothing (in case users do not see this warning message)

Reply:
The code now raises an exception when using `NCUObserver` when `nvmetrics` could not be loaded.